<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>299</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>snake_game settings</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="start_pushBtn">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>260</y>
      <width>99</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Start!</string>
    </property>
   </widget>
   <widget class="QFrame" name="randomWorld_frame">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>180</y>
      <width>381</width>
      <height>61</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::NoFrame</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Plain</enum>
    </property>
    <widget class="QLabel" name="width_label">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>10</y>
       <width>51</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Width</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="width_spinBox">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>61</width>
       <height>21</height>
      </rect>
     </property>
     <property name="minimum">
      <number>1</number>
     </property>
     <property name="maximum">
      <number>1000</number>
     </property>
     <property name="value">
      <number>50</number>
     </property>
    </widget>
    <widget class="QSpinBox" name="height_spinBox">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>10</y>
       <width>51</width>
       <height>21</height>
      </rect>
     </property>
     <property name="maximum">
      <number>1000</number>
     </property>
     <property name="value">
      <number>50</number>
     </property>
    </widget>
    <widget class="QLabel" name="height_label">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>10</y>
       <width>51</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Height</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="food_spinBox">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>61</width>
       <height>21</height>
      </rect>
     </property>
     <property name="maximum">
      <number>1000</number>
     </property>
     <property name="value">
      <number>5</number>
     </property>
    </widget>
    <widget class="QLabel" name="food_label">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>40</y>
       <width>41</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Food</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="bricks_spinBox">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>40</y>
       <width>51</width>
       <height>21</height>
      </rect>
     </property>
     <property name="minimum">
      <number>1</number>
     </property>
     <property name="maximum">
      <number>1000</number>
     </property>
    </widget>
    <widget class="QLabel" name="bricks_label">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>40</y>
       <width>41</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Bricks</string>
     </property>
    </widget>
   </widget>
   <widget class="QRadioButton" name="worldFile_radioBtn">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>121</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Use world file</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="randomWorld_radioBtn">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>150</y>
      <width>151</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Use random world</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
    <property name="autoRepeat">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>240</y>
      <width>401</width>
      <height>16</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLineEdit" name="command_lineEd">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>260</y>
      <width>261</width>
      <height>27</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="pathToExe_lineEd">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>251</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>snake_game.exe</string>
    </property>
   </widget>
   <widget class="QLabel" name="pathToExe_label">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>10</y>
      <width>111</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Path to exe file</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="snakeLength_spinBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
    <property name="maximum">
     <number>100000</number>
    </property>
    <property name="value">
     <number>1</number>
    </property>
   </widget>
   <widget class="QLabel" name="snakeLength_label">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>40</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Snake length</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="gameSpeed_spinBox">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>40</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
    <property name="maximum">
     <number>100000</number>
    </property>
    <property name="value">
     <number>20</number>
    </property>
   </widget>
   <widget class="QLabel" name="gameSpeed_label">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>40</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Game speed</string>
    </property>
   </widget>
   <widget class="QFrame" name="fileWorld_frame">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>100</y>
      <width>381</width>
      <height>41</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::NoFrame</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Plain</enum>
    </property>
    <widget class="QLabel" name="worldFilename_label">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>10</y>
       <width>111</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>World file name</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="worldFilename_lineEd">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>241</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>world.txt</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
